### PR TITLE
Update java_sqs.md

### PR DESCRIPTION
**Pull request by Nancyzhu278**
updating Java docs with SQS

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/data_streams/java.md [hunk 1]
```diff
@@ -35,10 +35,17 @@ As an alternative, you can set the `-Ddd.data.streams.enabled=true` system prope
 ```bash
 java -javaagent:/path/to/dd-java-agent.jar -Ddd.data.streams.enabled=true -jar path/to/your/app.jar
 ```
+### Libraries Supported
```
Comment (drichards-87) [inline content/en/data_streams/java.md:38]:
```suggestion
### Supported libraries
```

File: content/en/data_streams/java.md [hunk 2]
```diff
@@ -35,10 +35,17 @@ As an alternative, you can set the `-Ddd.data.streams.enabled=true` system prope
...
+Data Streams Monitoring supports the [confluent-kafka library][3].
+
+### Monitoring SQS Pipelines
```
Comment (drichards-87) [inline content/en/data_streams/java.md:41]:
```suggestion
### Monitoring SQS pipelines
```

File: content/en/data_streams/java.md [hunk 3]
```diff
@@ -35,10 +35,17 @@ As an alternative, you can set the `-Ddd.data.streams.enabled=true` system prope
...
+Data Streams Monitoring uses one [message attribute][4] to track a message's path through an SQS queue. As Amazon SQS has a maximum limit of 10 message attributes allowed per message, all messages streamed through the data pipelines must have 9 or less message attributes set, allowing the remaining attribute for Data Streams Monitoring.
```
Comment (drichards-87) [inline content/en/data_streams/java.md:42]:
```suggestion
Data Streams Monitoring uses one [message attribute][4] to track a message's path through an SQS queue. As Amazon SQS has a maximum limit of 10 message attributes allowed per message, all messages streamed through the data pipelines must have 9 or fewer message attributes set, allowing the remaining attribute for Data Streams Monitoring.
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some light feedback.

**Review by aliciascott — APPROVED** [review]

**Pull request merged**